### PR TITLE
upgrade to PHP_CodeSniffer 3

### PR DESCRIPTION
**Pull request by hboomsma**
* use namespaces
* fixed typos
* fixed setup

### CODE REVIEW

**Comment by hboomsma** [comment]
Built upon #15

File: phpcs.xml.dist [hunk 1]
```diff
@@ -1,15 +1,9 @@
 <?xml version="1.0"?>
 <ruleset name="Hostnet PHPCodeSniffer Standard">
   <description>It's a huge shame, but the PHPCodeSniffer Extension, does not follow it's own defined conding standard. But this is because the use of the PHPCS UnitTesting framework depending on certain class-names.</description>
```
Comment (nicoschoenmaker) [inline phpcs.xml.dist]: This comment is now out of date. PHPCodeSniffer now complies to it's own standard, but not to our additive standard (i.e. copyright version)

File: src/Hostnet/Sniffs/Functions/ReturnTypeDeclarationSniff.php [hunk 2]
```diff
@@ -181,23 +188,23 @@ private function getCharacterAfterReturnTypeDeclaration(PHP_CodeSniffer_File $ph
...
+    private function fixSpacing(File $phpcs_file, $tokens, $required_spacing, $start, $end)
     {
         //insert whitespace if there is no whitespace, and whitespace is required
         if (($start + 1) === $end && empty($required_spacing) === false) {
             $phpcs_file->fixer->addContent(
                 $start,
                 $required_spacing
             );
-        //otherwise, if there is whitespace, change the whitespace to the required spacing
+            //otherwise, if there is whitespace, change the whitespace to the required spacing
```
Comment (nicoschoenmaker) [inline src/Hostnet/Sniffs/Functions/ReturnTypeDeclarationSniff.php]: I think this comment should be one line lower, so it follows the `} else {` statement below.

**Comment by TomasVotruba** [comment]
Great work!

**Review by nicoschoenmaker — APPROVED** [review]

**Pull request merged**